.github/workflows/prebuild.yml

```diff
@@ -0,0 +1,44 @@
+name: "Prebuild checks"
+on:
+  pull_request:
```

Comment:
I am bit surprised this is a valid syntax without any letters follwing the `:`. Shouldn't it be
```
on:
  - pull_request
```
?

Reply:
Based on the [docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on)there seems to be multiple options but none is with - should be correct. Maybe it could be changed to ```
on: [pull_request]
```

Reply:
Ok maybe let's change it to that? Looks better to me :upside_down_face:

Reply:
Ok, I changed it to the `on: [pull_request]`.